docs/environments/atari.md

```diff
@@ -92,7 +92,7 @@ pip install supersuit
 
 Here is some example usage for the Atari preprocessing:
 
-``` python
+``` python notest
```

Comment:
SuperSuit issue I’m assuming, maybe for all of these documentation tests it’s possible to do pip install SuperSuit and shimmy and such, so these imports work? Not sure exactly how you’d do it, as I said above maybe fixtures or some other pytest feature.

Reply:
Got this to work on my branch thankfully